tests/integration/test_models_claude_openai_audio.py

```
@@ -955,14 +955,23 @@ def test_initialize_sets_initialized_flag(self) -> None:
         model.initialize()
         assert model._initialized is True
 
-    def test_generate_raises_not_implemented(self) -> None:
+    def test_generate_propagates_filenotfound_for_missing_audio(
+        self, tmp_path: pytest.TempPathFactory
```

Ruff (0.15.12)

[warning] Unused method argument: `tmp_path`

(ARG002)